out_writeapi.go

```diff
@@ -268,8 +263,9 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 	tableID := output.FLBPluginConfigKey(plugin, "TableID")
 
 	//optional maxchunksize param
-	maxChunkSize_init, err := getConfigField(plugin, "Max_Chunk_Size", "Invalid Max Chunk Size, defaulting to 9 MB", (9 * 1024 * 1024))
+	maxChunkSize_init, err := getConfigField(plugin, "Max_Chunk_Size", chunkSizeLimit)
 	if err != nil {
+		log.Printf("Invalid Max_Chunk_Size parameter in configuration file: %s", err)
```

Comment:
Do you know if the err message returned will include the faulty value that was found in the config file?

Reply:
No it won't - if we want to return the faulty value I can either include the print statement within the getConfigField function or add a third return value to the getConfigField and just include this faulty value in the print statement.